src/globus_cli/utils.py

```diff
@@ -0,0 +1,34 @@
+from typing import Dict
+
+
+def format_list_of_words(word: str, *other: str):
```

Comment:
`word`/`other` was a slightly confusing nomenclature to me, I think something `first`/`rest` might have been more apparent (since from the context it's already clear they're words)

Reply:
Sure, `first/rest` works for me.